src/sage/sets/set.py

```diff
@@ -1085,6 +1082,30 @@ def __richcmp__(self, other, op):
             return rich_to_bool(op, 0)
         return rich_to_bool(op, -1)
 
+    def isdisjoint(self, other):
```

Comment:
The standard in Sage is to add underscores between separate words in method names.

```suggestion
    def is_disjoint(self, other):
```

Reply:
I suppose fundamentally we should make a choice and then rename/deprecate. For the time being, implementing both could be alright (it's just an extra line `isdisjoint = is_disjoint` in the class).